articles/how-to-create-applications-for-azure-quantum.md

```diff
@@ -42,90 +36,80 @@ For example, this operation can NOT be run on a No Control Flow target:
-  
+      - IonQ QPU: `<ExecutionTarget>ionq.qpu</ExecutionTarget>`
+      - Honeywell API validator: `<ExecutionTarget>honeywell.hqs-lt-1.0-apival</ExecutionTarget>`
+      - Honeywell system model H0: `<ExecutionTarget>honeywell.hqs-lt-1.0</ExecutionTarget>`
+      - Honeywell system model H1: `<ExecutionTarget>honeywell.hqs-lt-s1</ExecutionTarget>`
+
    Your `*.csproj` file should look something like this:
-  
+
     ```xml
-    <Project Sdk="Microsoft.Quantum.Sdk/X.XX.XXXXXXXX">
-    
+    <Project Sdk="Microsoft.Quantum.Sdk/X.XX.XXXX.XXXXXX">
+
       <PropertyGroup>
         <OutputType>Exe</OutputType>
         <TargetFramework>netcoreapp3.1</TargetFramework>
-        <ExecutionTarget>ionq.qpu</ExecutionTarget>
+        <ExecutionTarget>my.target</ExecutionTarget>
       </PropertyGroup>
-    
+
     </Project>
     ```
 
-   where `X.XX.XXXXXXXX` is a place holder for the number of the latest version of the QDK.
-1. Write your Q# program, keeping in mind that you cannot compare measurement results to control the program flow. 
-1. Build and run your program locally using the QDK local
-   targets. These will let you know if your Q# application can run in IonQ's targets by checking the fulfillment of the No Control Flow restrictions and calculating the
-   needed resources.
-   - You can run your Q# program locally using the QDK full state simulator by
-     using the command `dotnet run`. Since you selected the `ExecutionTarget` in
-     the `*.csproj` file, the console output will warn you if you created a file that is not compatible with the No Control Flow profile.
-   - You can use the
-     [`resources estimator`](xref:microsoft.quantum.machines.overview.resources-estimator)
-     to estimate what resources your Q# program requires to run. You invoke the resources estimator with the command: `dotnet run --simulator
-     ResourcesEstimator`.
-1. Once your Q# program is ready, submit the job to Azure Quantum
-   using your preferred environment by using the target ID `ionq.qpu` for the QPU and `ionq.simulator` for the simulator.
+   where `X.XX.XXXX.XXXXXX` is a place holder for the latest version number of the QDK, and `my.target` a placeholder for your chosen target.
+1. Write your Q# program, keeping in mind the restrictions applying to the computation profile of your particular target.  you cannot compare measurement results to control the program flow.
+1. Build and run your program locally using the targets supplied by the QDK. This will let you know if your Q# application can run on the specified Azure Quantum target by checking the fulfillment of the computation profile restrictions and calculating the needed resources.
+   - You can run your Q# program locally using the QDK full state simulator by using the command `dotnet run`. Since you selected the `ExecutionTarget` in the `*.csproj` file, the console output will warn you if you created a file that is not compatible with selected computation profile.
```

Comment:
```suggestion
   - You can run your Q# program locally using the QDK full state simulator by using the command `dotnet run`. Since you selected the `ExecutionTarget` in the `*.csproj` file, the console output will warn you if you created a file that is not compatible with the selected computation profile.
```